packages/builder/src/components/common/FontAwesomeIcon.svelte

```diff
@@ -0,0 +1,15 @@
+<script context="module">
+  import { dom, library } from "@fortawesome/fontawesome-svg-core"
+  import {
+    faFileArrowUp,
+    faChevronLeft,
+  } from "@fortawesome/free-solid-svg-icons"
+  library.add(faFileArrowUp, faChevronLeft)
+  dom.watch()
+</script>
```

Comment:
I needed to add Font Awesome icons here also, this'll probably cause conflicts with the menu PR

Reply:
Yea in terms of this, I think we should merge the help menu PR, then update this one to fix any FA conflicts and (more importantly) fully revert any lockfile changes.